src/Maker/MakeRegistrationForm.php

```diff
@@ -313,15 +321,17 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
                     'user_class_name' => $userClassNameDetails->getShortName(),
                     'password_field' => $this->passwordField,
                     'will_verify_email' => $this->willVerifyEmail,
+                    'email_verifier_details' => new TemplateClassDetails($verifyEmailServiceClassNameDetails->getFullName(), $this->phpCompatUtil->canUseTypedProperties()),
```

Comment:
I don't like needing to pass that 2nd arg - it means me think that a factory might be better. What if we had a `$generator->createClassHelper(string $fqcn): TemplateClassHelper` method?

I also changed `Details` to `Helper`... that's subjective :)

Reply:
Also, maybe we call the variable `email_verifier_class`? I know that makes it sound like a string, but it's more descriptive than "details". And, we could add a `__toString()` if we wanted to, which would return the fcqn.

Reply:
I think I strayed away from this initially as the `Generator` is not internal. But adding an `@experimental` tag to the method takes care of that. Love it and done.